index.html

```diff
@@ -107,5 +108,9 @@ <h2 class="d-md-none fw-bold">Your direct access to Noroff's best candidates</h2
   </main>
   <footer></footer>
 </body>
-
+</html>
+      </section>
+    </main>
+    <footer></footer>
+  </body>
 </html>
```

Comment:
There are still some issues here that needs to be resolved. There seems to be additional section, main, footer, body and html closing tags